src/components/application_manager/test/commands/command_request_impl_test.cc

```diff
@@ -402,8 +411,13 @@ TEST_F(CommandRequestImplTest, CheckAllowedParameters_MsgParamsMap_SUCCESS) {
 }
 
 TEST_F(CommandRequestImplTest, AddDisallowedParameters_SUCCESS) {
-  const std::string kDisallowedParam =
-      ::application_manager::MessageHelper::vehicle_data().begin()->first;
+  const std::string kDisallowedParam = "disallowed_param";
```

Comment:
@LevchenkoS Move all the parameters like this out of class into nameless namespace.